Documentation/Cluster.rst

```diff
@@ -69,3 +71,68 @@ To the remove a node from the cluster, run:
     
     sudo mcvirt cluster remove-node --node <Remote Node Name>
     
+
+Get Cluster information
+-----------------------
+
+* In order to view status information about the cluster, use the 'info' parameter for MCVirt, without specifying a VM name:

```

Comment:
Use the double colon in line at the end of the paragraph, see https://en.wikipedia.org/wiki/ReStructuredText#Literal_blocks

same elsewhere